source/intro_to/explore_software.rst

```diff
@@ -0,0 +1,229 @@
+
+:bash:`bet` is the Brain Extraction Tool, which segments the brain from non-brain in structural and functional data and models skull and scalp surfaces.
+
+:bash:`fast` is FMRIB's Automated Segmentation Tool, which provides brain segmentation (into different tissue types) and bias field correction.
+
+:bash:`flirt` is FMRIB's Linear Image Registration Tool, which offers linear inter- and intra-modal registration.
+
+:bash:`fnirt` is FMRIB's NonLinear Image Registration Tool, which offers linear inter- and intra-modal registration.
+
+:bash:`siena` is a structural brain change analysis tool used for estimating brain atrophy.
+
+:bash:`susan` is a nonlinear noise reduction tool.
+
+Diffusion MRI
+"""""""""""""""""""""""
+:bash:`bedpostx` is a tool in FSL that uses Bayesian estimation with Markov Chain Monte Carlo sampling to model diffusion MRI signal as fiber orientation distributions and estimate diffusion parameters at each voxel for local modeling of diffusion parameters, including estimation of the number and orientation of fiber bundles.
+
+:bash:`probtrackx` is a tool in FSL that performs tractography and connectivity-based segmentation using probabilistic tractography. It calculates the probability of connection between pairs of voxels in the brain by simulating diffusion-weighted MRI signal propagation along different possible fiber pathways to investigate the connectivity of different brain regions and segment the brain into functional networks based on connectivity patterns.
+
+:bash:`eddy` is a tool that performs eddy current correction and motion correction for diffusion MRI data.
+
+:bash:`topup` is a tool that corrects for susceptibility-induced distortions in diffusion MRI data.
+
+:bash:`xtract` (cross-species tractography) is a tool that automatically extract a set of tracts in humans and macaques. It can also be used to define one's own tractography protocols where all the user needs to do is to define a set of masks in standard space (e.g. MNI152).
+
+**TBSS** is Tract-Based Spatial Statistics, part of FMRIB's Diffusion Toolbox, which offers voxelwise analysis of multi-subject diffusion data.
+
+Installation
+^^^^^^^^^^^^^^^^^^^^^
+
+Visit this website for download and install instructions as well as a general overview of tools available in the package `(here) <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki>`__
+
+You will have to download a Python file install (after accepting a license) and then install FSL (by executing the Python installer)
+`(here) <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation>`__
+
+It is common to add these lines to your :bash:`.bashrc` to make FSL easier to use;
+
+.. code-block:: bash
+
+    # export FSLDIR=/PATH/TO/FSL
+    . ${FSLDIR}/etc/fslconf/fsl.sh
+    export PATH=${FSLDIR}/bin:${PATH}
+
+Once installed, you should be able to type :bash:`bet` or :bash:`fast` to see the help display. :bash:`fsleyes` will launch the visualization tool.
+
+
+ANTs
+-----------------------
```

Comment:
On my computer, when I write something like this, there is a warning that the number of -- should be the same length as the title.
```
ANTS
----
```

not

```
ANTS
---------------------
```

I never checked if it is important...